### PR TITLE
issue #205015 Telemetry Response Event is not capturing properly for …

### PR DESCRIPTION
**Pull request by ajinkyapandetekdi**
…Configurable Telemetry

### CODE REVIEW

File: src/components/Game.js [hunk 1]
```diff
@@ -64,6 +66,82 @@ function Game() {
...
     checkVoice(voiceText, Story[storyLine]);
+    generateResponseEvent(voiceText)
+  }
+
+
+  function generateResponseEvent(voiceText){
+   
+    let texttemp = voiceText.toLowerCase();
+    const studentTextArray = texttemp.split(" ");
+    let tempteacherText = localStorage.getItem("contentText")?.toLowerCase();
```
Comment (gouravmore) [inline src/components/Game.js:77]: use Story[storyLine] value instead of localstorage value

File: src/components/Game.js [hunk 2]
```diff
@@ -64,6 +66,82 @@ function Game() {
...
+    let result_per_words = 0;
+    let result_per_words1 = 0;
+    let occuracy_percentage = 0;
+
+    let word_result_array = compareArrays(
+      teacherTextArray,
+      studentTextArray
+    );
+
+    for (let i = 0; i < studentTextArray?.length; i++) {
```
Comment (gouravmore) [inline src/components/Game.js:96]: looks like this logic is already implemented in checkVoice(). Please check it and reuse the code

File: src/components/Game.js [hunk 3]
```diff
@@ -116,6 +194,7 @@ function Game() {
     }
     let temp = storyLine + 1;
     setStoryLine(temp);
+    localStorage.setItem("storyline",temp);
```
Comment (gouravmore) [inline src/components/Game.js:197]: if this is not required - remove it

**Review by gouravmore — CHANGES_REQUESTED** [review]

File: src/components/Game.js (hunk 1 above)
Comment (ajinkyapandetekdi) [reply src/components/Game.js]: Removed

File: src/components/Game.js (hunk 3 above)
Comment (ajinkyapandetekdi) [reply src/components/Game.js]: Removed

**Pull request merged**